README.md

```diff
@@ -26,14 +26,23 @@ Temporal can be configured to run with a couple of database choices.
 
 #### Default Installation: Batteries Included
 
-By default, Temporal Helm Chart configures Temporal to runs with Cassandra (for persistence) and ElasticSearch (for "visibility" features). By default, Temporal Helm Chart installs all the dependencies, out of the box.
+By default, Temporal Helm Chart configures Temporal to runs with Cassandra (for persistence) and ElasticSearch/Kafka (for "visibility" features), Prometheus, and Grafana. By default, Temporal Helm Chart installs all dependencies, out of the box.
 
 To install Temporal with all of its dependencies, including Cassandra and ElasticSearch, run this command:
 
 ```bash
 ~/temporal-helm$ helm install temporaltest . --timeout 900s
 ```
 
+To use your own instance of ElasticSearch, MySQL, or Cassandra, please read the "Bring Your Own" sections below.
+
+Other components can be omitted by setting their corresponding 'enable' flag to `false`, e. g.
+
+```bash
+~/temporal-helm$ helm install --set grafana.enabled=false temporaltest . --timeout 900s
```

Comment:
This looks slightly confusing (at lease to me). `enabled=false` means "disable". One can get impression that after this command his graphana cluster will be disabled. Or at least grphana features in Temporal will be disabled. `graphana.install` looks like a better name for me.

Reply:
It might be applicable to other PRs also.

Reply:
oh that's a good point. i think this is a bit of a convention in in helm world to use `enabled` to indicate that the thing is omitted from installation. i'll think on this a bit, to see if there is a better way.